app/src/main/res/values-v21/styles.xml

```diff
@@ -4,7 +4,6 @@
     <style name="wsAppTheme" parent="Theme.AppCompat.Light.NoActionBar">
         <item name="colorPrimary">@color/primaryColor</item>
         <item name="colorPrimaryDark">@color/primaryColorDark</item>
-        <item name="statusBarColor">@color/primaryColorDark</item>
```

Comment:
"statusBarColor" doesn't seem to be a valid item name and throws a compilation error. I think this worked before because the folder was incorrectly named "values-21" and was likely not parsed. The app looks fine without this line.